exo-run/features/steps/steps-given.ls

```diff
@@ -18,4 +19,4 @@ module.exports = ->
                                        cwd: @app-dir,
                                        stdout: dim-console.process.stdout
                                        stderr: dim-console.process.stderr)
-      done!
+      wait 10_000, done
```

Comment:
Docker Compose gives us output that contains the output of each service, multiplexed. If we create a parser for it's output, we can de-multiplex it and create a bunch of individual streams.